### PR TITLE
implement homework9

### PR DESCRIPTION
**Pull request by dctenna**

### CODE REVIEW

File: 09_algorithms_and_data_structures_2/main.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { mockedAdjacencyMatrix } from './mocked-graph-data';
+import { WeightedGraph } from './WeightedGraph';
+
+const graph = new WeightedGraph(mockedAdjacencyMatrix);
```
Comment (AlexHladin) [inline 09_algorithms_and_data_structures_2/main.ts:4]: @dctenna, the main goal is to create an application that can create the weighted graph and keep it as the adjacency matrix. You should create the adjacency matrix based on the graph.

**Review by AlexHladin — CHANGES_REQUESTED** [review]

File: 09_algorithms_and_data_structures_2/main.ts (hunk 1 above)
Comment (dctenna) [reply 09_algorithms_and_data_structures_2/main.ts:26]: @AlexHladin done

Comment (AlexHladin) [reply 09_algorithms_and_data_structures_2/main.ts:26]: @dctenna, but where are you constructing adjacency matrix? I see that you have moched adjacency matrix?

File: 09_algorithms_and_data_structures_2/mocked-graph-data.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export const mockedAdjacencyMatrix = [
```
Comment (AlexHladin) [inline 09_algorithms_and_data_structures_2/mocked-graph-data.ts:1]: @dctenna, what for the mockedAdjacencyMatrix? You need to create the matrix itself. And it should look like on the screenshot
image

**Review by AlexHladin — CHANGES_REQUESTED** [review]

File: 09_algorithms_and_data_structures_2/main.ts (hunk 1 above)
Comment (dctenna) [reply 09_algorithms_and_data_structures_2/main.ts:26]: @AlexHladin I am constructing it here: 
Mocked matrix is the same as constructed to show that we can construct any matrix.

image

**Pull request merged**